### ISSUE TITLE
Suggestion: calculation for transferred sizes

### ISSUE THREAD

**Issue by ts-thomas** [krausest/js-framework-benchmark#1597]
I think the section "Transferred size (in kBs)" isn't really helpful actually because of result calculation based on 2Kb vanilla code. In the result table, the color code for this specific tests isn't helpful. I suggest to apply some kind of ![image](https://github.com/krausest/js-framework-benchmark/assets/12898005/545ccbe3-c352-4cec-bd47-303657e2868f) for the result calculation for this section. What do you think?

**Comment by ts-thomas** [comment]
@atellmer I like this idea 👍

**Comment by krausest** [comment]
I'm not sure that I can follow. Where's the issue?
![Screenshot 2024-02-06 at 19 54 38](https://github.com/krausest/js-framework-benchmark/assets/3374055/b4a87297-ab0b-470c-aa27-0e67d90af82f)
I find it amazing that some implementations can beat vanillajs(-1) regarding compressed size.
I understand that for this metric many implementations are > 4 times as big. 
However I like the idea that the range of the color scale is the same for all benchmarks (i.e. full red means >4 times as "bad" as the best implementation).

**Comment by atellmer** [comment]
I agree that the idea of a reference implementation is a good one. But I believe that the standard is well suited when we evaluate speed, but not entirely suitable when we evaluate the weight of libraries. This criterion turns out to be too strict, because we evaluate only relative indicators and do not take into account absolute ones. As a result, color codes lose their semantic component in the vast majority of implementations. For example, Svelte and Inferno weigh quite little in brotli from a real world point of view, but from the point of view of the benchmark approach they are put in the category with Angular and React, which weigh “a lot”.
![1](https://github.com/krausest/js-framework-benchmark/assets/16635118/838ebf87-8b36-4df1-8b59-27e90294f5a8)

**Comment by krausest** [comment]
I'm still not convinced. There are 17 implementations that are (including their framework) smaller than vanillajs.
But maybe I could undo leaving out the css and fonts in the calculation. This would add about 145 kb which would reduce the spread as you want but it would allow me to keep the color scale for the factors constant for all benchmarks. Should we try that?

**Comment by atellmer** [comment]
To be honest, I'm not sure that including styles and fonts in the bundle will help us in assessing the weight of implementations. It's probably better to leave everything as it is for now, because it's still important to see dry js.
